### ISSUE TITLE
Bus error with node-sass@3.0.0

### ISSUE THREAD

**Issue by josip** [sass/node-sass#946]
I'm having a problem compiling a couple of files with `node-sass@3.0.0`, the error I get doesn't seem very helpful unfortunately:

```
$ node-sass src/scss/app.scss
[1]    65321 bus error  node-sass src/scss/app.scss
```

I can compile the same file with both and `node-sass@2.x` and latest `sassc`.

The root of the problem seems to be this `@import` statement:

```scss
@import '../../node_modules/x/vendor/bootstrap/bootstrap';
```
([Bootstrap](https://github.com/twbs/bootstrap-sass/tree/master/assets/stylesheets))

Using node 0.10.32 on OS X 10.10.4.

**Comment by xzyfer** [comment]
Hi @josip, this issue has come a few times but noone has given up a workable test cases.

This is issue is very rare and hard to reproduce. If you are able to strip down your code to and make it available to us that would be great. Unfortunately we cannot fix this issue if we cannot reproduce it.

**Comment by josip** [comment]
@xzyfer http://cl.ly/3U0x0F1b0C1P if you run `test.sh` you should see it fail with `node-sass`.

For comparison, this is what I get:
```
$ ./test.sh
node-sass	3.0.0	(Wrapper)	[JavaScript]
libsass  	3.2.2	(Sass Compiler)	[C/C++]
./test.sh: line 5: 68412 Bus error: 10           node-sass src/scss/app.scss
=================
sassc: 3.1.0
libsass: 3.0.3
sass2scss: 1.0.3
/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
```

**Comment by xzyfer** [comment]
Is the issue present with a more recent scss?

```
brew update && brew install sassc
```

**Comment by josip** [comment]
ah brew bottle is outdated.

Fresh from source:
```
$ sassc --version
sassc: 3.2.1
libsass: 3.2.1
sass2scss: 1.0.3
                                                                                                                               
$ sassc src/scss/app.scss
[1]    87021 segmentation fault  sassc src/scss/app.scss
```

do you want me to reopen this in sassc/libsass repo?

**Comment by xzyfer** [comment]
I was able to reproduce this issue from the code you supplied. I was able to track it down to this line from `bootstrap/mixins/_hide-text.scss`

```scss
font: #{0/0} a;
```

This has been fixed in the recent Libsass 3.2.3 (https://github.com/sass/libsass/issues/1162) and will be the upcoming node-sass 3.0.1.

I'll test locally on node-sass master to confirm this issue is resolved.

**Comment by xzyfer** [comment]
Confirming this is resolved in node-sass@master.

**Comment by xzyfer** [comment]
Fixed in 3.1.0.
